extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -30,18 +22,25 @@ amp-sidebar::part(sidebar) {
   background-color: inherit;
   height: inherit;
   top: inherit;
-  max-height: inherit;
-  max-width: inherit;
-  min-width: inherit;
```

Comment:
Just confirming that these styles are also not required for `amp-sidebar::part(c)` and `amp-sidebar::part(wrapper)`?

Reply:
Thats correct. The sidebar still renders as expected without those styles for `part(c)` and `part(wrapper)`.